### DISCUSSION TITLE
Average reflex time

### DISCUSSION THREAD

**Discussion by paulik123** [jonathan-s/django-sockpuppet#130] [Q&A]
Hi, what is a normal reflex rountrip time? Because I don't know what to aim for. I'm currently at 200ms 300ms for a medium sized page. Is that ok? Or should I aim for lower?

**Comment by jonathan-s** [comment]
First make it work, then make it fast. I wouldn't worry too much until you'll ready to ship the website. That said, installing lxml will make the parsing faster (it's using beautifulsoup by default). It's also possible to make partial [dom updates](https://sockpuppet.argpar.se/scoping#partial-dom-updates).

**Reply by paulik123** [reply]
Premature optimization is the root of all evil, I know, but I was curious what's an average roundtrip time.

**Comment by paulik123** [comment]
Is there a way to use both AlpineJS for purely UI actions that don't require server interaction(like opening a modal) and Django sockpuppet? Sort of like Liveview does it?
